### PR TITLE
[12.0] [ADD] hr_timesheet_time_type: Ability to add time type in timesheet lines.

### PR DESCRIPTION
**Pull request by Chandresh-SerpentCS**
- Project / Configuration menu option, to configure "Time Type" **(For example, "Internal Meeting", "Design", "Development", "Tests")**
- On Timesheet lines, "Time Type" column

### CODE REVIEW

**Comment by alexey-pelykh** [comment]
It seems highly similar to what have been discussed here https://github.com/OCA/timesheet/issues/354 - is that related? cc @dreispt

**Comment by Chandresh-SerpentCS** [comment]
@alexey-pelykh Yes, It is the solution for #354.

**Comment by Chandresh-SerpentCS** [comment]
@dreispt I would like to request you for reviewing this PR.

File: hr_timesheet_time_type/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+{
+    "name": "Time Type in Timesheet",
+    "summary": "Ability to add time type in timesheet lines.",
+    "version": "12.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "Open Source Integrators, Odoo Community Association (OCA)",
+    "category": "Timesheet",
+    "website": "https://github.com/OCA/timesheet",
+    "depends": ["hr_timesheet", "project"],
```
Comment (alexey-pelykh) [inline hr_timesheet_time_type/__manifest__.py]: `hr_timesheet` already depends on `project`, no need to specify

**Review by alexey-pelykh — CHANGES_REQUESTED** [review]

**Review by dreispt — APPROVED** [review]
LGTM, code review

File: hr_timesheet_time_type/__manifest__.py (hunk 1 above)
Comment (dreispt) [inline hr_timesheet_time_type/__manifest__.py]:
```suggestion
    "depends": ["hr_timesheet"],
```

**Comment by dreispt** [comment]
@alexey-pelykh Your comments were addressed.

**Review by alexey-pelykh — APPROVED** [review]

**Pull request merged**